packages/turf-isolines/index.js

```diff
@@ -1,28 +1,27 @@
-//https://github.com/jasondavies/conrec.js
-//http://stackoverflow.com/questions/263305/drawing-a-topographical-map
-var tin = require('@turf/tin');
 var bbox = require('@turf/bbox');
-var grid = require('@turf/point-grid');
-var inside = require('@turf/inside');
-var square = require('@turf/square');
+var meta = require('@turf/meta');
 var helpers = require('@turf/helpers');
-var distance = require('@turf/distance');
-var planepoint = require('@turf/planepoint');
-var featurecollection = helpers.featureCollection;
-var linestring = helpers.lineString;
-var point = helpers.point;
-var Conrec = require('./conrec');
+var invariant = require('@turf/invariant');
+var gridToMatrix = require('grid-to-matrix');
+var marchingsquares = require('marchingsquares');
+var multiLineString = helpers.multiLineString;
+var coordEach = meta.coordEach;
+var collectionOf = invariant.collectionOf;
+var featureCollection = helpers.featureCollection;
 
 /**
  * Takes {@link Point|points} with z-values and an array of
  * value breaks and generates [isolines](http://en.wikipedia.org/wiki/Isoline).
  *
  * @name isolines
  * @param {FeatureCollection<Point>} points input points
- * @param {string} z the property name in `points` from which z-values will be pulled
- * @param {number} resolution resolution of the underlying grid
- * @param {Array<number>} breaks where to draw contours
- * @returns {FeatureCollection<LineString>} isolines
+ * @param {Array<number>} breaks where to draw isolines
+ * @param {string} [zProperty='elevation'] the property name in `points` from which z-values will be pulled
+ * @param {Object} [options={}] options on output
```

Comment:
>  (would break backwards compatibility)

Well actually compatibility is already an issue, since the `resolution` parameter is not used/useful anymore (it is defined by the `pointGrid` input itself).
Maybe we keep this module update for the next major release?

> there's very rare amounts of TurfJS modules that use this type of syntax

We'll need to update `turf-isoband` as well then, as it follow this same syntax?
This would be a breaking change as well.